src/Components/Facility/PatientNoteCard.tsx

```diff
@@ -0,0 +1,37 @@
+import { relativeDate, formatDateTime, classNames } from "../../Utils/utils";
+import { USER_TYPES_MAP } from "../../Common/constants";
+
+const PatientNoteCard = ({ note }: { note: any }) => {
+  return (
+    <div
+      key={note.id}
+      className={classNames(
+        "mt-4 flex w-full flex-col rounded-lg border border-gray-300 bg-white p-3 text-gray-800",
+        note.user_type === "RemoteSpecialist" && "border-primary-400"
+      )}
+    >
+      <div className="flex">
+        <span className="text-sm font-semibold text-gray-700">
+          {note.created_by_object?.first_name || "Unknown"}{" "}
+          {note.created_by_object?.last_name}
+        </span>
+        <span className="pl-2 text-sm text-gray-700">
+          {"("}
+          {USER_TYPES_MAP[note.user_type]}
+          {")"}
```

Comment:
Prefer using string interpolation here.
```suggestion
          {`(${USER_TYPES_MAP[note.user_type]})`}
```